src/logic/community/broadcaster.ts

```diff
@@ -96,18 +116,49 @@ export function createCommunityBroadcasterComponent(
     }
   }
 
+  /**
+   * Publishes an event to all moderators and owners of a community.
+   * @param event - The event to publish.
+   * @returns A promise that resolves when the event is published.
+   */
+  async function publishToOwnersAndModerators(
+    event: CommunityRequestToJoinReceivedEventReducedMetadata
+  ): Promise<void> {
+    const moderatorsAndOwners: string[] = await getAllCommunityMembersAddresses(event.metadata.communityId, {
+      roles: [CommunityRole.Moderator, CommunityRole.Owner]
+    })
+
+    await sns.publishMessage({
+      type: Events.Type.COMMUNITY,
+      subType: event.subType,
+      key: event.key,
+      timestamp: event.timestamp,
+      metadata: {
+        ...event.metadata,
+        addressesToNotify: moderatorsAndOwners
+      }
+    } as CommunityRequestToJoinReceivedEvent)
+  }
+
   async function broadcast(
     event:
       | CommunityDeletedEventReducedMetadata
       | CommunityRenamedEventReducedMetadata
       | CommunityMemberRemovedEvent
       | CommunityMemberBannedEvent
+      | CommunityRequestToJoinAcceptedEvent
+      | CommunityRequestToJoinReceivedEventReducedMetadata
+      | CommunityInviteReceivedEvent
   ) {
     const shouldReportEventToAllMembers =
       event.subType === Events.SubType.Community.DELETED || event.subType === Events.SubType.Community.RENAMED
 
+    const shouldReportEventToOwnersAndModerators = event.subType === Events.SubType.Community.REQUEST_TO_JOIN_RECEIVED
+
     if (shouldReportEventToAllMembers) {
       await publishToAllMembers(event as CommunityDeletedEventReducedMetadata | CommunityRenamedEventReducedMetadata)
+    } else if (shouldReportEventToOwnersAndModerators) {
+      await publishToOwnersAndModerators(event as CommunityRequestToJoinReceivedEventReducedMetadata)
     } else {
       await sns.publishMessage(event)
     }
```

Comment:
WDYT about having a dictionary with the different "publishing strategies" so we can decide what strategy to use for any specific event subtype? The function would become cleaner, and we won't need to keep adding new variables `shouldReport...` and if-else branches.

Reply:
Will apply this feedback in a further PR!